py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py

```diff
@@ -556,3 +731,17 @@ def le(
         lt(ctx, target, source_ir, f"{name}_lt", lhs_val, rhs_val),
         eq(ctx, target, source_ir, f"{name}_eq", lhs_val, rhs_val),
     )
+
+
+def get_value_by_condition(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    other: TRTTensor,
+    condition: TRTTensor,
+) -> TRTTensor:
+    select_layer = ctx.net.add_select(condition, input, other)
+    set_layer_name(select_layer, target, name + "_select", source_ir)
+    return select_layer.get_output(0)
```

Comment:
This can be removed - replace with usage of `impl.condition.select`, from here:
https://github.com/pytorch/TensorRT/blob/76c9ebdc7544f7f666b63e21d13abfc9e2d44c5d/py/torch_tensorrt/dynamo/conversion/impl/condition/ops.py#L102-L113

Reply:
Alternatively, you may need `where`:
https://github.com/pytorch/TensorRT/blob/76c9ebdc7544f7f666b63e21d13abfc9e2d44c5d/py/torch_tensorrt/dynamo/conversion/impl/condition/ops.py#L18-L26

Reply:
I removed `get_value_by_condition` and replaced it with `impl.condition.select.`